pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/AccessorMethodGenerationRule.java

```diff
@@ -57,6 +67,30 @@ public void analyzeMember(final AbstractJavaAccessNode node, final List<NameOccu
             return;
         }
 
+        for (final ASTVariableDeclarator varDecl: node.findChildrenOfType(ASTVariableDeclarator.class)) {
+            if (varDecl.hasInitializer()) {
```

Comment:
we should also check if the var is final, otherwise, even with an initializer, the compiler will never inline it. We are also missing a test case for this scenario.